src/main/groovy/org/transmartproject/batch/highdim/datastd/StandardDataValuePatientInjectionProcessor.groovy

```diff
@@ -0,0 +1,43 @@
+package org.transmartproject.batch.highdim.datastd
+
+import org.springframework.batch.item.ItemProcessor
+import org.springframework.beans.factory.annotation.Autowired
+import org.transmartproject.batch.highdim.assays.AssayMappingsRowStore
+import org.transmartproject.batch.highdim.assays.MappingFileRow
+import org.transmartproject.batch.patient.Patient
+import org.transmartproject.batch.patient.PatientSet
+
+/**
+ * Set the patient to the @link StandardDataValue bean.
+ */
+class StandardDataValuePatientInjectionProcessor implements ItemProcessor<StandardDataValue, StandardDataValue> {
+
+    @Autowired
+    private AssayMappingsRowStore assayMappings
+
+    @Autowired
+    private PatientSet patientSet
+
+    private final Map samplePatientMap = [:]
```

Comment:
Is this simply a cache? If so, please mark it as such. If it's not, you'll have to implement `ItemStream`.